api/scpca_portal/email_templates/dataset_ready.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Your dataset is ready!</title>
+    <style>
```

Comment:
According to [Can I email](https://www.caniemail.com/features/html-style/), some mail clients (e.g., Gmail Mobile) remove the first `<head>`, so it's recommended to place `<style>` inside a second `<head>`. To be safe, using inline CSS is the safest approach.